app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java

```diff
@@ -483,9 +485,11 @@ void testAutomatedClaimOutOfScope() {
    * message, Slack message and removal of RDR1 special issue are verified.
    */
   @Test
+  @SneakyThrows
   void testAutomatedClaimMissingAnchor() {
     AutomatedClaimTestSpec spec = new AutomatedClaimTestSpec("20");
     spec.setPayloadPath("test-mas/claim-20-7101-noanchor.json");
+    spec.setExpectedStatusCode(HttpStatus.UNPROCESSABLE_ENTITY);
     spec.setExpectedMessage(
```

Comment:
This is not an off-ramp case anymore. Please remove the last line in this method (testAutomatedClaimOffRamp(spec)). That should have been failing.